### ISSUE TITLE
layer begin and end positions are invalid!

### ISSUE THREAD

**Issue by TSL-RamKrishna** [isovic/racon#118]
I am getting an error

[racon::Window::add_layer] error: layer begin and end positions are invalid!

I ran racon twice and I get the same error. Any help  or suggestion?

**Comment by rvaser** [comment]
Hello,
which racon version are you using? Could you provide the data that is causing the error?

Best regards,
Robert

**Comment by TSL-RamKrishna** [comment]
I am using version 1.2.0. The data that I am using is very huge (5.1GB compressed).
I aligned the reads using minimap. The output SAM output is then the input for racon. I am doing minimap-racon twice. In the first round, racon completed successfully. I get this error in the second round.

**Comment by rvaser** [comment]
Could you please try the second iteration with the latest racon version (`1.3.3`)?

**Comment by TSL-RamKrishna** [comment]
Here some more lines just before the error.

[racon::Polisher::initialize] loaded target sequences
[racon::Polisher::initialize] loaded sequences
[racon::Polisher::initialize] loaded overlaps
[racon::Polisher::initialize] aligned overlap 668099/668099
[racon::Window::add_layer] error: layer begin and end positions are invalid!

**Comment by TSL-RamKrishna** [comment]
sure. I will try with the latest racon version and let you know. Thanks

**Comment by TSL-RamKrishna** [comment]
I think the latest release is 1.3.2. I could not find v1.3.3 in github releases.

**Comment by rvaser** [comment]
Versions `1.3.3` (latest commit on master, but not a release) and `1.3.2` are almost the same, the newer one has a smaller log while runnig.

Best regards,
Robert

**Comment by TSL-RamKrishna** [comment]
We just found out that our genome assembly has a duplicated contig name. Can this duplicated naming error cause this error?

**Comment by rvaser** [comment]
Probably! Did you manage to run the newer version? Any errors there?

**Comment by TSL-RamKrishna** [comment]
Nope. Have not got chance to run. I just found out at there is a duplicated contig name this morning. I got new assembly without duplicated contigname. I will try with this new assembly now. Our HPC has v1.3.2 installed. I am thinking to test with v1.3.2 for now as , according to you, there is not much difference between v1.3.2 and v1.3.3, except the log filesize.

**Comment by rvaser** [comment]
Sure, go ahead.

**Comment by TSL-RamKrishna** [comment]
Racon ran successfully now. I ran racon-1.3.2 and with the no contig name duplication. Thanks for the help.
